student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(age, name = 'Unknown', classroom: '')
+    super(name, age)
```

Comment:
- [ ] Please pass the `parent_permission` parameter to the `initialize` method which is then passed to the `super` method.